sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala

```diff
@@ -295,6 +295,41 @@ case class AlterViewAsCommand(
   }
 }
 
+/**
+ * A command for users to get views in the given database.
+ * If a databaseName is not given, the current database will be used.
+ * The syntax of using this command in SQL is:
+ * {{{
+ *   SHOW VIEWS [(IN|FROM) database_name] [[LIKE] 'identifier_with_wildcards'];
+ * }}}
+ */
+case class ShowViewsCommand(
+    databaseName: Option[String],
+    tableIdentifierPattern: Option[String]) extends RunnableCommand {
+
+  // The result of SHOW VIEWS has three basic columns: database, viewName and isTemporary.
+  override val output: Seq[Attribute] = Seq(
+    AttributeReference("database", StringType, nullable = false)(),
```

Comment:
shall we follow the new abstraction and call it `namespace`?

Reply:
Currently `VIEW` related commands are only supported in `v1 SessionCatalog`, maybe we can keep it along with V1 abstraction? The `SHOW TABLES` will resolve to v2 `ShowTablesExec` which use attrName `namespace`, while in the v1 version `ShowTablesCommand` it still using attrName `database`.  WDYT @cloud-fan , I'm OK with both decision  :-)

Reply:
Updated in d14b4fa698873cf0eef4f1ca50b9a501c5cd868e, thanks!